VotingApplication/VotingApplication.Web/Scripts/Services/ManageService.js

```diff
@@ -0,0 +1,53 @@
+﻿(function () {
+    angular.module('GVA.Creation').factory('ManageService', ['$location', '$http', '$routeParams', function ($location, $http, $routeParams) {
```

Comment:
This is rather a long line. Would be a bit easier on the eyes if .module and .controller were each on their own line.